_docs/src/documentation/rex_pipeline.md

```diff
@@ -148,16 +178,16 @@ This script launches clang-tidy and clang-format if clang tools are enabled. Bec
 Some Examples:
 ```sh
 # Builds regina project and its dependencies in debug mode using msvc compiler
-py _build.py -project=regina -config=debug -comp=msvc
+py _rex.py build -project=regina -config=debug -compiler=msvc
 
 # Cleans all regina and its depedencies' intermediate files and then performs a build afterwards
-py _build.py -project=regina -config=debug -comp=msvc -clean
+py _rex.py build -project=regina -config=debug -compiler=msvc -clean
 ```
```

Comment:
These seem to be examples on how to trigger a build and have nothing to do with a post build event ( except that it gets triggered? )